content/blog/2025/08/2025-08-14-project-hydra-improving-state-res.md

```diff
@@ -9,15 +9,15 @@ category = ["General", "Security"]
 
 Hi all,
 
-On July 16th 2025 we issued a [pre-disclosure](https://matrix.org/blog/2025/07/security-predisclosure/) for vulnerabilities in the federation protocol, and [announced new releases](https://matrix.org/blog/2025/08/security-release/) of Matrix homeservers on Mon August 11\. Today we are ending the embargo and disclosing the remaining MSCs. This post will go into more detail about the changes and what led up to them.
+On July 16th 2025 we issued a [pre-disclosure](@/blog/2025/07/security-predisclosure/2025-07-16-security-predisclosure.md) for vulnerabilities in the federation protocol, and [announced new releases](@/blog/2025/08/security-release/2025-08-11-security-release.md) of Matrix homeservers on Mon August 11. Today we are ending the embargo and disclosing the remaining MSCs. This post will go into more detail about the changes and what led up to them.
```

Comment:
the path to the first file is [content/blog/2025/07/2025-07-16-security-predisclosure.md](https://github.com/matrix-org/matrix.org/blob/main/content/blog/2025/07/2025-07-16-security-predisclosure.md) ->
```suggestion
On July 16th 2025 we issued a [pre-disclosure](@/blog/2025/07/2025-07-16-security-predisclosure.md) for vulnerabilities in the federation protocol, and [announced new releases](@/blog/2025/08/security-release/2025-08-11-security-release.md) of Matrix homeservers on Mon August 11. Today we are ending the embargo and disclosing the remaining MSCs. This post will go into more detail about the changes and what led up to them.
```
etc :)